src/components/search/SearchResults.jsx

```diff
@@ -73,14 +86,25 @@ const SearchResults = ({
 
   const resultsHeading = useMemo(
     () => {
-      const resultsLabel = nbHits === 0 || nbHits > 1 ? 'results' : 'result';
+      const resultsLabel = nbHits === 0 || nbHits > 1
+        ? intl.formatMessage({
+          id: 'enterprise.search.page.content.results',
+          defaultMessage: 'Results',
+          description: 'Label for the search results count when we have more than one result.',
+        })
+        : intl.formatMessage({
+          id: 'enterprise.search.page.content.result',
+          defaultMessage: 'Result',
+          description: 'Label for the search result count when we have only one result.',
```

Comment:
It would be great if we can combine these two `formatMessage` into a single using singular and plural

Reply:
it won't be easy to handle the undefined case so I will not apply singular and plural format.